lightbluetent/home.py

```diff
@@ -9,23 +9,44 @@
 import ucam_webauth.raven
 import ucam_webauth.raven.flask_glue
 
-ILLEGAL_NAME_RE = re.compile(r'[:,=\n]')
-ILLEGAL_NAME_ERR = "Please do not use any of the following characters: : , = ⏎ "
-
 bp = Blueprint("home", __name__)
 
 auth_decorator = ucam_webauth.raven.flask_glue.AuthDecorator(desc="SRCF Lightbluetent")
 
 @bp.route("/")
 def index():
-    return render_template("home/index.html")
+    return render_template("home/index.html", page_title="Welcome to the 2020 Virtual Freshers' Fair!")
 
 @bp.route("/logout")
 def logout():
     auth_decorator.logout()
-    flash("You have been logged out.")
     return redirect(url_for("home.index"))
 
+@bp.route("/log_in")
+@auth_decorator
+def log_in():
+    return redirect(url_for("home.home"))
+
+@bp.route("/home")
+@auth_decorator
```

Comment:
We willl run into a problem with `auth_decorator` here, since the homepage will be put behind Raven.  I'm guessing this line should be gone?

Reply:
As clarified in chat: /home is the page where a list of societies associated with the authenticated user is displayed. We're using / for the 'public' landing page.